packages/financial-templates-lib/src/price-feed/BalancerPriceFeed.js

```diff
@@ -18,7 +18,7 @@ class BalancerPriceFeed extends PriceFeedInterface {
     lookback,
     twapLength,
     poolDecimals = 18,
-    decimals = 18
+    priceFeedDecimals = 18
```

Comment:
This was a nasty bug: because this construction was named `decimals` and not `priceFeedDecimals`, then `DefaultPriceFeedConfigs` was not correctly overriding this precision.